### DISCUSSION TITLE
Student Developer Pack issue

### DISCUSSION THREAD

**Discussion by kcprjblnsk** [community/community#109961] [GitHub Education]
### Hi everyone,

Hello,
my student developer pack got both approved and rejected. I didn't have filled billing information but somehow I got approved.
It was like 4 months ago but together with Microsoft Support(because I can't get access to Azure pack due to rejection) we found an issue. How can I fix it? I already added billing information but I can't reverify(blocked option), rest of required options like 2FA is active.

![github xddd](https://github.com/community/community/assets/116565697/c5370be1-24a0-4578-9209-d1f8c1d04b54)

**Comment by Prajapati992** [comment]
I am facing the same issue 😔
